src/tests/topp/MapAlignerPoseClustering_1_trafo2.trafoXML

```diff
@@ -2,12 +2,12 @@
 <TrafoXML version="1.0" xsi:noNamespaceSchemaLocation="http://open-ms.sourceforge.net/schemas/TrafoXML_1_0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 	<Transformation name="linear">
 		<Param  type="string" name="symmetric_regression" value="false"/>
-		<Param  type="string" name="x_weight" value=""/>
+		<!-- <Param  type="string" name="x_weight" value=""/>
```

Comment:
hmm I am not sure if this is the correct way to handle this. Wouldn't it make sense to omit these parameter if no weighting functions are specified?